pkgs/top-level/all-packages.nix

```
@@ -40805,4 +40805,16 @@ with pkgs;
   reindeer = callPackage ../development/tools/reindeer { };
 
   charasay = callPackage ../tools/misc/charasay { };
+
+  libvoikko = callPackage ../development/libraries/libvoikko/default.nix { };
+
+  hfst = callPackage ../development/libraries/hfst/hfst.nix { };
+
+  hfst-python = python3Packages.callPackage ../development/libraries/hfst/hfst-python.nix { };
+
+  hfst-ospell = callPackage ../applications/misc/hfst-ospell/default.nix { };
+
+  omorfi = callPackage ../applications/misc/omorfi/omorfi.nix { };
```

Please rename omorfi.nix to default.nix
```suggestion
  omorfi = callPackage ../applications/misc/omorfi { };
```